src/components/attendee/AttendeeHeader/AttendeeHeader.module.scss

```diff
@@ -3,10 +3,13 @@
 @use "scss/attendee/space";
 @use "scss/attendee/layer";
 
+$-z-index--container: layer.above(0);
+$-z-index--header: layer.above($-z-index--container);
```

Comment:
These are module private, the container layer is basis for this module, all the rest can be steps upward or downward relative to eachother